conftest.py

```diff
@@ -26,6 +28,13 @@ def pytest_addoption(parser):
                      type=str,
                      help='Set GCP project to test. Required for GCP tests.')
 
+    # While only used for Heroku at the moment, GitHub tests are soon to be
+    # added, which will also need an "organization" option. Current plan is to
+    # reuse this one.
+    parser.addoption('--organization',
```

Comment:
let's call this `--heroku-organization`

Reply:
Why not use the generic? Both GitHub & Heroku need that as a prameter. Are we envisioning a future where one pytest invocation tests everything?

Reply:
As discussed, separate github and heroku args for consistency